player/lua/console.lua

```diff
@@ -1990,9 +1990,16 @@ mp.register_script_message('get-input', function (script_name, args)
 
     if args.items then
         selectable_items = {}
+
+        -- Limit the number of characters to prevent libass from freezing mpv.
+        -- Not important for terminal output.
+        local limit = terminal_output() and 5000
+                      or 5 * mp.get_property_native('osd-width') / opts.font_size
+
         for i, item in ipairs(args.items) do
-            selectable_items[i] = item:gsub("[\r\n].*", "⋯"):sub(1, 300)
+            selectable_items[i] = item:gsub("[\r\n].*", "⋯"):sub(1, limit)
```

Comment:
Should we add `⋯` when it is cut by the limit?

Reply:
The limit should not be visibile, we want to go beyond the window width and let it be clipped.